lambdas/service/app.py

```diff
@@ -653,9 +653,9 @@ def validate_repository_search(params, **validators):
         'catalog': validate_catalog,
         'filters': validate_filters,
         'order': validate_order,
-        'search_after': str,
+        'search_after': validate_search_after,
```

Comment:
```suggestion
        'search_after': partial(validate_json_param, 'search_before')
```

using `functools.partial`